### PR TITLE
Update monitor cmdlet warning message

### PR DESCRIPTION
**Pull request by lijinpei2008**
Update monitor cmdlet warning message

https://github.com/Azure/azure-powershell/issues/17013#issuecomment-1034655155

### CODE REVIEW

**Review by dingmeng-xue — DISMISSED** [review]

**Comment by dingmeng-xue** [comment]
Change log is required.

File: src/Monitor/Monitor/MonitorCmdletBase.cs [hunk 1]
```diff
@@ -67,13 +67,19 @@ protected void WriteIdentifiedWarning(string cmdletName, string topic, string me
                 topic,
                 message);
 
-            if (withTimeStamp)
-            {
-                WriteWarningWithTimestamp(formattedMessage);
-            }
-            else
+            string supressWarningOrErrorValue = System.Environment.GetEnvironmentVariable(BreakingChangeAttributeHelper.SUPPRESS_ERROR_OR_WARNING_MESSAGE_ENV_VARIABLE_NAME);
```
Comment (VeryEarly) [inline src/Monitor/Monitor/MonitorCmdletBase.cs:70]: please check line 96 - 101, please remove the duplicate logic.

Comment (lijinpei2008) [reply src/Monitor/Monitor/MonitorCmdletBase.cs:63]: Done, please check it.

File: src/Monitor/Monitor/MonitorCmdletBase.cs [hunk 2]
```diff
@@ -60,20 +60,34 @@ protected string GetCmdletName()
...
+            bool supressWarningOrError;
+            Boolean.TryParse(supressWarningOrErrorValue, out supressWarningOrError);
+            if (!supressWarningOrError)
             {
-                WriteWarning(formattedMessage);
+                if (topic == "ExecuteCmdlet")
```
Comment (VeryEarly) [inline src/Monitor/Monitor/MonitorCmdletBase.cs:68]:
```suggestion
                string formattedMessage = string.Format(
                CultureInfo.InvariantCulture,
                "{0}{1}{2}",
                string.IsNullOrEmpty(cmdletName)?cmdletName:"["cmdletName"] ",
                string.IsNullOrEmpty(topic)?topic:topic": ",
                message);

            if (withTimeStamp)
            {
                WriteWarningWithTimestamp(formattedMessage);
            }
            else
            {
                WriteWarning(formattedMessage);
            }
```

File: src/Monitor/Monitor/MonitorCmdletBase.cs [hunk 3]
```diff
@@ -87,16 +101,12 @@ public override void ExecuteCmdlet()
...
             try
             {
-                bool supressWarningOrError;
-                Boolean.TryParse(supressWarningOrErrorValue, out supressWarningOrError);
-                if (!supressWarningOrError)
-                {
-                    WriteWarningWithTimestamp("The namespace for all the model classes will change from Microsoft.Azure.Management.Monitor.Management.Models to Microsoft.Azure.Management.Monitor.Models in future releases.");
-                    WriteWarningWithTimestamp("The namespace for output classes will be uniform for all classes in future releases to make it independent of modifications in the model classes.");
-                }
+                this.WriteIdentifiedWarning(
```
Comment (VeryEarly) [inline src/Monitor/Monitor/MonitorCmdletBase.cs:106]: WriteIdentifiedWarning("The namespace for all the model classes will change from Microsoft.Azure.Management.Monitor.Management.Models to Microsoft.Azure.Management.Monitor.Models in future releases.");
WriteIdentifiedWarning("The namespace for output classes will be uniform for all classes in future releases to make it independent of modifications in the model classes.");

**Review by VeryEarly — CHANGES_REQUESTED** [review]
please polish code

File: src/Monitor/Monitor/MonitorCmdletBase.cs (hunk 3 above)
Comment (lijinpei2008) [reply src/Monitor/Monitor/MonitorCmdletBase.cs:96]: Done.

**Comment by lijinpei2008** [comment]
> please polish code

Done

**Comment by dingmeng-xue** [comment]
It is good to me.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**